source/_layouts/page-pattern.html

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+---
+<div id="main" class="col-md-12 clearfix" role="main">
+  <article class="post">
+    <header class="page-tabs-header">
+      <div class="row">
+        <div class="col-sm-12 col-md-6 cols-nopad-right">
+          <h1>{{ page.title }}</h1>
+        </div>
+        <div class="col-sm-12 col-md-6 cols-nopad-left cols-nopad-right">
+          {% if page.code == false %}
+            {% include nav-tabs-pattern.html codetab=false %}
+          {% elsif page.design == false %}
+            {% include nav-tabs-pattern.html designtab=false %}
+          {% else %}
+            {% include nav-tabs-pattern.html %}
+          {% endif %}
+        </div>
+      </div>
+    </header>
+    <section class="post_content">
+      <div class="tab-content">
+        <div role="tabpanel" class="tab-pane active" id="overview">
```

Comment:
I know there is an overview.md file for heat map, but this may be a bad example to follow? On pf.org, heat map does not have any examples on the overview page; however, this is the exception -- there isn't an example for core patternfly, only an angular implementation.

Every other overview page on pf.org has examples, included from patternfly. This is in addition to the examples for the code tab.

Perhaps the designers decided to remove the examples from overview? Or, the heat map is a bad example to follow?